src/niswitch/examples/niswitch_channel_properties.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+
+import argparse
+import niswitch
+import sys
+
+parser = argparse.ArgumentParser(description='TODO: write something.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+args = parser.parse_args()
+
+try:
+    with niswitch.Session(args.name) as session:
+        with session.channel("Marcos") as chan:
```

Comment:
Which switch has a "Marcos" channel?

Reply:
This was written for local testing and I didn't mean to commit. I don't think the NI-SWITCH examples are that interesting for channel-based stuff so I'm leaning towards removing. We should write something good for NI-DCPower.
Thoughts?

Reply:
I think that is reasonable. So remove this example

Reply:
I have removed this example and expanded src/niswitch/examples/niswitch_get_device_info.py